server/job.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+job_completed = False
```

Comment:
Do you like it `global`? Job could be a class. It has sense, `Storage` is a class too.

Everything else, great, :+1: